x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/alerts_list.ts

```diff
@@ -373,15 +373,19 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
           '[data-test-subj="P50Percentile"]:nth-of-type(1)'
         );
         const percentileCellText = await percentileCell.getVisibleText();
-        expect(percentileCellText).to.match(/\d{2,}:\d{2}/);
+        if (percentileCellText !== 'N/A') {
```

Comment:
instead of doing that, let's just do that `.toMatch(/^N\/A|\d{2,}:\d{2}$/);`, I prefer that because like that we are saying that these two values are valid and not ignoring it.